### PR TITLE
Fix and uniformize verbose

### PR DESCRIPTION
**Pull request by karanphil**
# Quick description
Second attempt at uniformizing the verbose option. I added this line: `logging.getLogger().setLevel(logging.getLevelName(args.verbose))` to all scripts, and modified the way the verbose argument is used (in scilpy.io.utils). I also cleaned a bit some modules, so when there was only debug logging, they are now info logging.

There might be some scripts like scil_dwi_compute_snr.py that used logging.info to print some results, but with the new default as warning, they won't work properly without -v. I don't know all the script in scilpy so I would need some help to know if I broke anything. :)
...

## Type of change

Check the relevant options.

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Provide data, screenshots, command line to test (if relevant)

...

# Checklist

- [ ] My code follows the style guidelines of this project (run [autopep8](https://pypi.org/project/autopep8/))
- [ ] I added relevant citations to scripts, modules and functions docstrings and descriptions
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I moved all functions from the script file (except the argparser and main) to scilpy modules
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by karanphil** [comment]
Note to myself: Once PR 904 is merged, remove the verbose argument from dwi.operations.detect_volume_outliers and replace print by logging.info.

File: scripts/scil_tractogram_filter_by_orientation.py [hunk 1]
```diff
@@ -85,19 +85,16 @@ def _build_arg_parser():
 
 
 def main():
-
     parser = _build_arg_parser()
     args = parser.parse_args()
+    logging.getLogger().setLevel(logging.getLevelName(args.verbose))
+    # Silencing SFT's logger if our logging is in DEBUG mode, because it
+    # typically produces a lot of outputs!
+    set_sft_logger_level('WARNING')
```
Comment (arnaudbore) [inline scripts/scil_tractogram_filter_by_orientation.py:93]: you should keep the if condition here if ask for DEBUG then set sft to WARNING

File: scripts/scil_tractogram_filter_by_roi.py [hunk 2]
```diff
@@ -237,24 +237,22 @@ def check_overwrite_distance(parser, args):
 def main():
     parser = _build_arg_parser()
     args = parser.parse_args()
+    logging.getLogger().setLevel(logging.getLevelName(args.verbose))
+    set_sft_logger_level('WARNING')
```
Comment (arnaudbore) [inline scripts/scil_tractogram_filter_by_roi.py:241]: same here

File: scripts/scil_tractogram_resample.py [hunk 3]
```diff
@@ -116,6 +116,8 @@ def _build_arg_parser():
 def main():
     parser = _build_arg_parser()
     args = parser.parse_args()
+    logging.getLogger().setLevel(logging.getLevelName(args.verbose))
+    set_sft_logger_level(args.verbose)
```
Comment (arnaudbore) [inline scripts/scil_tractogram_resample.py:120]: same here

File: scripts/scil_tractogram_segment_bundles_for_connectivity.py [hunk 4]
```diff
@@ -243,6 +243,9 @@ def _build_arg_parser():
 def main():
     parser = _build_arg_parser()
     args = parser.parse_args()
+    logging.getLogger().setLevel(logging.getLevelName(args.verbose))
+    coloredlogs.install(level=logging.getLevelName(args.verbose))
+    set_sft_logger_level('WARNING')
```
Comment (arnaudbore) [inline scripts/scil_tractogram_segment_bundles_for_connectivity.py:248]: same here

File: scripts/scil_tractogram_split.py [hunk 5]
```diff
@@ -83,6 +83,8 @@ def _build_arg_parser():
 def main():
     parser = _build_arg_parser()
     args = parser.parse_args()
+    logging.getLogger().setLevel(logging.getLevelName(args.verbose))
+    set_sft_logger_level(args.verbose)
```
Comment (arnaudbore) [inline scripts/scil_tractogram_split.py:87]: don't really understand this one.

File: scripts/scil_tractogram_filter_by_anatomy.py [hunk 6]
```diff
@@ -551,8 +549,8 @@ def main():
                 save_tractogram(sft, rejected_sft_name)
             return
 
-        logging.debug('The file {} contains 0 streamlines after '.format(
-            out_sft_name) + step + ' filtering')
+        logging.info('The file {} contains 0 streamlines after '.format(
+                     out_sft_name) + step + ' filtering')
         save_tractogram(new_sft, out_sft_name)
 
     if args.verbose:
```
Comment (arnaudbore) [inline scripts/scil_tractogram_filter_by_anatomy.py:556]: if verbose > warning

**Review by arnaudbore — APPROVED** [review]
Almost there ! :) Godd job !

**Pull request merged**